src/zcl_abapgit_environment.clas.abap

```diff
@@ -0,0 +1,36 @@
+CLASS zcl_abapgit_environment DEFINITION
+  PUBLIC
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    CLASS-METHODS is_cloud
+      RETURNING
+        VALUE(rv_cloud) TYPE abap_bool .
+  PROTECTED SECTION.
+
+    CLASS-DATA gv_cloud TYPE abap_bool VALUE abap_undefined ##NO_TEXT.
+  PRIVATE SECTION.
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_ENVIRONMENT IMPLEMENTATION.
+
+
+  METHOD is_cloud.
```

Comment:
In case of we would like to use a variable which has more two values we need to switch to a case statement.